packages/groq/package.json

```diff
@@ -1,6 +1,7 @@
 {
+  "private": true,
+  "type": "commonjs",
   "name": "groq",
-  "private": "true",
```

Comment:
I changed this from `"private": "true"` to `"private": true`. But seems like this should be removed instead?

Reply:
not sure what the intention was behind marking it as private, but looks like it could have been a temporary measure (cc @rexxars)

Reply:
I removed `"private": true` – which is hardly what we want :-)